static/js/src/static-forms.js

```diff
@@ -185,3 +186,60 @@ const forms = document.querySelectorAll("form[action='/marketo/submit']");
 if (forms.length) {
   forms.forEach((form) => setUpStaticForms(form, extractMarketoID(form.id)));
 }
+
+/**
+ *
+ * @param {*} fieldset
+ * @param {*} checklistItem
+ *
+ * Disable & enable Ubuntu versions checklist based on user selection
+ */
+function ubuntuVersionsChecklist(fieldset, checklistItem) {
```

Comment:
Looks good! but I would make it even more generic, there might be a case where another form wants to use the same pattern